index.html

```diff
@@ -199,7 +199,7 @@ <h3>## Diversity</h3>
         <br />
 
         <p>
-          We'll inform you as soon as the application phase for diversity tickets will begin.
+          The application phase started shortly after the ticket sale. Use the <a target= "_blank" href = "https://docs.google.com/forms/d/e/1FAIpQLSdRY_j_snQky14Szoe9M3CztqQ4_AJcaovt8ebF3Px2a8-H_g/viewform">diversity form</a> to apply. The application phase will end at 2020/03/27 and applicants will be contacted before 2020/04/01.
```

Comment:
maybe just `The application phase is live. Use the ...` ?

Reply:
or has started?